Application/Source/Service/LoginService.swift

```diff
@@ -55,8 +55,10 @@ final class LoginService {
 
     func logout() -> Observable<Result<Void, FirebaseCommonError>> {
         do {
-            try FIRAuth.auth()?.signOut()
-            return .just(.success())
+            try Auth.auth().signOut()
+            //return .just(Result.success(FirebaseCommonError.appNotAuthorized))
+            let firebaseError = FirebaseCommonError(error: NSError())
+            return .just(.failure(firebaseError))
```

Comment:
Why are you returning `.failure` here? Shouldn't it be successful if the `try` didn't throw?